Docs/developer_guide/extensions.md

```diff
@@ -213,6 +214,12 @@ adding a playback button using [this free service](https://addplaybuttontoimage.
   - Create a pull request: by clicking ''Create pull request'' button
   - Follow the instructions in the pull request template
 
+:::{tip}
+Once your extension is successfully integrated into the Extensions Index, you gain access to daily build and test reports for verification. For additional information, refer to [](#extension-continuous-integration).
+
+While it is possible to develop Python-based extensions without a [local extension build](#extension-build), creating one can significantly expedite the development cycle. This allows you to promptly confirm that the extension can be properly packaged and distributed, providing a quicker feedback loop compared to waiting for the daily build and test reports.
```

Comment:
The link for `[local extension build](#extension-build)` doesn’t appear to be working. Nothing seems to happen when I click on it as though there is an incorrect anchor.

Reply:
Thanks for the review :pray: This should now be fixed.